### DISCUSSION TITLE
Welcome to agwa Discussions!

### DISCUSSION THREAD

**Discussion by haiyan-Wei** [ARS-SWRC/agwa#4] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jburgessconforti** [comment]
Hi Haiyan,

Fire season is going to start ramping up shortly and I was wondering if there have been any updates to AGWA-Pro? There were some stability issues on the fires I tried to use it on late last year and am hoping I can get this incorporated into our workflow this year.

**Comment by haiyan-Wei** [comment]
Hi,

Thank you for reaching out. The latest version of AGWA-Pro is managed and
updated on GitHub:
https://github.com/ARS-SWRC/agwa

I recommend downloading the most recent code and running through the fire
assessment tutorial, if you haven’t done so. Since Shea moved on to a
different position, I’ve been working on the programming and updates.

If you run into any issues, please let me know—I'd be happy to help
troubleshoot or look into any specific problems. You can also reach me
directly at ***@***.***

Wishing you a smooth fire season ahead!


Best,

Haiyan
